PluginsAndFeatures/azure-toolkit-for-intellij/src/main/java/com/microsoft/intellij/FileStore.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for license information.
+ */
+package com.microsoft.intellij;
+
+import com.microsoft.azure.toolkit.ide.common.store.IFileStore;
+
+import javax.annotation.Nonnull;
+import javax.annotation.Nullable;
+
+public class FileStore implements IFileStore {
+
+    @Nullable
+    @Override
+    public String getProperty(@Nonnull String s, @Nonnull String s1) {
```

Comment:
Please also update the parameter name as well

Reply:
Will impl it in later PR